README.md

```diff
@@ -1,61 +1,97 @@
 DirectPV
 ----------
 
-![build](https://github.com/minio/directpv/workflows/Go/badge.svg) ![license](https://img.shields.io/badge/license-AGPL%20V3-blue) 
-
-DirectPV is a [CSI driver](https://kubernetes-csi.github.io/docs/introduction.html) for dynamically provisioning direct persistent volumes. 
+DirectPV is a CSI driver for [Direct Attached Storage](https://en.wikipedia.org/wiki/Direct-attached_storage). In a simpler sense, it is a distributed persistent volume manager, and not a storage system like SAN or NAS. It is useful to *discover, format, mount, schedule and monitor* drives across servers. Since Kubernetes `hostPath` and `local` PV are statically provisioned and limited in functionality, DirectPV was created to address this limitation. 
 
```

Comment:
```suggestion

DirectPV was previously named DirectCSI and is available as an equivalent binary starting with the 2.0.0 release. Kubernetes deployments using older DirectCSI versions should install DirectPV as part of their upgrade to 2.0.0+. See
the [Upgrades](#upgrade) for instructions on upgrading to DirectPV.

```

Nice to have this up top to catch users up. Looking at the releases,

Reply:
We are not replacing direct-csi by directpv. We are instead launching directPV for the first time now. Think of  direct-csi as a temporary working name we were using so far